example/spec/parallel/startSF/startSfSpec.js

```diff
@@ -274,94 +238,25 @@ describe('the sf-starter lambda function', () => {
       ]);
     });
 
-    it('queue-granules returns the correct amount of queued executions', async () => {
```

Comment:
I decided that queuing the executions via queue-granules wasn't really essential for what this test is trying to prove. This test should only care that for any execution messages queued with the necessary values, the executions are limited to the max for the queue.

This is also why I updated the unit tests for the various lambdas that add messages to the queues (queue-granules, queue-pdrs, sf-scheduler) to ensure that they all preserve the necessary message attributes for the execution throttling to happen.